package.json

```diff
@@ -67,7 +69,11 @@
     "transform": {
       "^.+\\.(js|jsx)$": "babel-jest",
       "^.+\\.(ts|tsx)$": "ts-jest"
-    }
+    },
+    "testMatch": [
+      "**/__tests__/**/*.[jt]s?(x)",
+      "**/__integrations__/**/*.integration.[jt]s?(x)"
+    ]
```

Comment:
Great, I was about to raise this issue as I just added a new test and I noticed that it was not being executed.